jfbrandhorst/grpc-gateway-build-env image can't run on Windows

The `jfbrandhorst/grpc-gateway-build-env:1.13` image can't be pulled in a Windows docker environment:


```powershell
PS C:\> docker pull jfbrandhorst/grpc-gateway-build-env:1.13
Error response from daemon: mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not ''
```
And running it as described in the contributing doc from a linux environment errors with:

```bash
protoc -I /usr/local/bin//../include -I. -Ithird_party/googleapis --plugin=bin/protoc-gen-swagger --swagger_out=logtostderr=true,allow_repeated_fields_in_body=true,use_go_templates=true,Mgoogle/protobuf/field_mask.proto=google.golang.org/genproto/protobuf/field_mask,Mgoogle/protobuf/descriptor.proto=github.com/golang/protobuf/protoc-gen-go/descriptor,Mexamples/proto/sub/message.proto=github.com/grpc-ecosystem/grpc-gateway/examples/proto/sub,grpc_api_configuration=examples/proto/examplepb/unannotated_echo_service.yaml:. examples/proto/examplepb/echo_service.proto examples/proto/examplepb/a_bit_of_everything.proto examples/proto/examplepb/wrappers.proto examples/proto/examplepb/stream.proto examples/proto/examplepb/unannotated_echo_service.proto examples/proto/examplepb/use_go_template.proto examples/proto/examplepb/response_body_service.proto
generate -i examples/proto/examplepb/echo_service.swagger.json \
    -l go -o examples/clients/echo --additional-properties packageName=echo
make: generate: Command not found
make: *** [Makefile:169: examples/clients/echo/client.go] Error 127
```

_Originally posted by @someone1 in https://github.com/grpc-ecosystem/grpc-gateway/pull/1072#issuecomment-548820266_

--- a/.circleci/Dockerfile
+++ b/.circleci/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.13.1
+FROM golang:1.13.4
 
 # Warm apt cache and install dependencies
 # bzip2 is required by the node_tests (to extract its dependencies).
@@ -17,7 +17,7 @@ RUN echo -e '#!/bin/bash\njava -jar /usr/local/bin/swagger-codegen-cli.jar "$@"'
 	chmod +x /usr/local/bin/swagger-codegen
 
 # Install protoc
-ENV PROTOC_VERSION=3.9.2
+ENV PROTOC_VERSION=3.10.1
 RUN wget https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip \
     -O /protoc-${PROTOC_VERSION}-linux-x86_64.zip && \
     unzip /protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local/ && \

--- a/.circleci/README.md
+++ b/.circleci/README.md
@@ -10,7 +10,7 @@ Great, it should be as simple as thus (run from the root of the directory):
 $ docker run -v $(pwd):/go/src/github.com/grpc-ecosystem/grpc-gateway --rm jfbrandhorst/grpc-gateway-build-env:1.13 \
     /bin/bash -c 'cd /go/src/github.com/grpc-ecosystem/grpc-gateway && \
         make realclean && \
-        make examples SWAGGER_CODEGEN="${SWAGGER_CODEGEN}"'
+        make examples'
 ```
 
 If this has resulted in some file changes in the repo, please ensure you check those in with your merge request.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -27,7 +27,7 @@ Great, it should be as simple as thus (run from the root of the directory):
 docker run -v $(pwd):/src/grpc-gateway --rm jfbrandhorst/grpc-gateway-build-env:1.13 \
     /bin/bash -c 'cd /src/grpc-gateway && \
         make realclean && \
-        make examples SWAGGER_CODEGEN="${SWAGGER_CODEGEN}"'
+        make examples'
 docker run -itv $(pwd):/grpc-gateway -w /grpc-gateway --entrypoint /bin/bash --rm \
     l.gcr.io/google/bazel -c 'bazel run :gazelle -- update-repos -from_file=go.mod -to_macro=repositories.bzl%go_repositories; bazel run :buildifier'
 ```

